addons/mail/security/ir.model.access.csv

```diff
@@ -69,3 +69,5 @@ ir_actions_report_access_user,ir.actions.report.access.user,base.model_ir_action
 access_mail_link_preview_admin,mail.link.preview.admin,model_mail_link_preview,base.group_erp_manager,1,1,1,1
 access_discuss_gif_favorite,discuss.gif.favorite,model_discuss_gif_favorite,base.group_user,1,1,1,1
 access_discuss_voice_metadata_user,discuss.voice.metadata.user,model_discuss_voice_metadata,,0,0,0,0
+access_mail_message_translation,mail.message.translation,model_mail_message_translation,base.group_user,1,1,1,1
+access_mail_message_translation_portal,mail.message.translation.portal,model_mail_message_translation,base.group_portal,1,1,1,0

```

Comment:
I would suggest to set everything to 0 and just place one or two correct sudo after ensuring message can be read.

Reply:
Seems to be a more robust approach, thanks!